content/rdi/quickstart/write-behind-guide.md

```diff
@@ -10,15 +10,17 @@ draft:
 hidden: false
 ---
 
-This guide takes you through the creation of a write-behind pipeline.
+This guide takes you through the creation of a write-behind and read-through pipelines.
 
-> Note: Write Behind is currently in Preview.
+> Note: write-behind & read-through are currently in Preview.
 
 ## Concepts
 
-**Write-behind**: An RDI policy and pipeline to synchronize the data in a Redis DB with some downstream data store.
+**Write-behind**: RDI pipeline to synchronize the data in a Redis DB with some downstream data store.
 You can think about it as a pipeline that starts with change data capture (CDC) events for a Redis database and then filters, transforms, and maps the data to the target data store data structure.
 
+**Read-through**: RDI pipeline to automatically fetch data from a downstream data store back to Redis cache in case of cache-miss. This read-through is initiated by client attempt to retrieve a key from Redis that does not exist. RDI will retrieve the key, store it in the cache and then provide a synchronous response to the client.
```

Comment:
maybe include also that we can manipulate the data as well.

Reply:
done